### DISCUSSION TITLE
[IonTable] Create copy value from table cell

### DISCUSSION THREAD

**Discussion by iurynogueira** [iurynogueira/ion#476] [Ideas]
We can create a functionality to copy to clipboard the value of table cell when click in icon or in value.

**Comment by iurynogueira** [comment]
@andersonnagelo @Jeter-Megaron @qrayannes This is good to user? What do you think?

**Reply by Jeter-Megaron** [reply]
Interesting, will be very good in other components like the indicator. We just need to define how it will work at the design level.

**Reply by andersonnagelo** [reply]
It would have the option to copy the value to the clipboard, in cases that need it, right? I find it interesting, it could appear on mouse hover, it's a possibility, but you can leave it fixed too, like icon button

**Comment by mizek1** [comment]
This could be a directive to be added to any element, not only table cells, since is a feature that could be needed on other components. It could be exported from ion to be used on projects.

**Comment by allanplobo** [comment]
I think it's an interesting feature, we could pass a parameter to say what value, for example the ID, a name, an address, depending on the use it can be very nice to have this native in the ion table or following mizek1 idea.

Which solution to implement? I prefer to wait what the designers think of this feature as being general or not.

**Reply by iurynogueira** [reply]
I believe that we can create a directive and call in table when the dev inform a specific config on cell

**Reply by iurynogueira** [reply]
> I believe that we can create a directive and call in table when the dev inform a specific config on cell

@mizek1 @allanplobo this make sense for you?

**Reply by mizek1** [reply]
Sure! We just need to check how it would work, if will be on components click or if we will display a button after the value to be copied, like @andersonnagelo mentioned above.

**Reply by iurynogueira** [reply]
Nice, so we are waiting designers decision.
